components/carousel/module/CarouselAgnostic.vue

```diff
@@ -142,15 +148,11 @@ if (props.config) {
     { immediate: true },
   )
 
-  watchDebounced(
-    [() => props.config, () => props.items.length],
-    () => {
-      if (slider.value) {
-        slider.value.update(props.config)
-      }
-    },
-    { debounce: SYNC_CONFIG_DEBOUNCE_AMOUNT, immediate: true },
-  )
+  onMounted(updateConfig)
+
+  watchDebounced([() => props.config, () => props.items.length], updateConfig, {
+    debounce: SYNC_CONFIG_DEBOUNCE_AMOUNT,
+  })
```

Comment:
simpler?

```  suggestion
watchDebounced([() => props.config, () => props.items.length], () => slider.value?.update(props.config), {
    debounce: SYNC_CONFIG_DEBOUNCE_AMOUNT,
  })
  
  ```
  
  
  and delete `updateConfig` function

Reply:
yep, done